Makefile.core.mk

```diff
@@ -703,6 +703,20 @@ show.goenv: ; $(info $(H) go environment...)
 show.%: ; $(info $* $(H) $($*))
 	$(Q) true
 
+#-----------------------------------------------------------------------------
+# Target: custom resource definitions
+#-----------------------------------------------------------------------------
+
+API_TMP := $(shell mktemp -d -u)/api
+API_UPDATE_BRANCH ?= "master"
+
+update-crds:
+	@mkdir -p $(API_TMP)
+	@git clone -q --depth 1 --single-branch --branch $(API_UPDATE_BRANCH) https://github.com/istio/api $(API_TMP)
+	@rm -f crd-all.gen.yaml
```

Comment:
doesn't this need the installer/kubernetes/... prefix?

Reply:
Yes. Thanks! Added.